src/smc-webapp/account/store.ts

```diff
@@ -39,7 +39,7 @@ export class AccountStore extends Store<AccountState> {
     return this.get("account_id");
   }
 
-  selectors: any = {
+  selectors = {
```

Comment:
Woohoo!  selectors are correctly typed based on the `State` in `Store<State>` now!